.gitignore

```diff
@@ -134,4 +134,4 @@ dist
 # Auto-generated HTML files
 docs/
 apidocs/
-devdocs/
+devdocs/
```

Comment:
minor np: You've removed the newline at the end of this file, try changing your editor to add newlines at the end of the file and save again, or fix manually.